egs2/wsj/asr1/README.md

```diff
@@ -28,6 +28,29 @@
 |decode_asr_whisper_noctc_greedy_asr_model_valid.acc.ave/test_dev93|503|46683|99.4|0.3|0.3|0.2|0.8|24.9|
 |decode_asr_whisper_noctc_greedy_asr_model_valid.acc.ave/test_eval92|333|32096|99.6|0.2|0.2|0.2|0.6|19.5|
 
+## Self-supervised learning features [WavLm_large, Conformer, utt_mvn](conf/tuning/train_asr_conformer_s3prlfrontend_wavlm.yaml) with [Transformer-LM](conf/tuning/train_lm_transformer.yaml)
+
+## Environments
+- date: `Tue Jul 11 05:34:44 UTC 2023`
+- python version: `3.7.4 (default, Aug 13 2019, 20:35:49)  [GCC 7.3.0]`
+- espnet version: `espnet 202304`
+- pytorch version: `pytorch 1.10.1+cu111`
+- Git hash: `a0702913d1ce828915abb4638abcb308d85400d4`
+  - Commit date: `Sun Aug 13 08:04:03 2023 +0900`
+
+### WER
+
+|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+|---|---|---|---|---|---|---|---|---|
+|decode_lm_lm_train_lm_transformer_en_char_valid.loss.ave_asr_model_valid.acc.ave/test_dev93|503|8234|97.7|2.1|0.3|0.2|2.6|28.0|
+|decode_lm_lm_train_lm_transformer_en_char_valid.loss.ave_asr_model_valid.acc.ave/test_eval92|333|5643|98.6|1.3|0.1|0.1|1.5|19.5|
+
+### CER
+
+|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+|---|---|---|---|---|---|---|---|---|
+|decode_lm_lm_train_lm_transformer_en_char_valid.loss.ave_asr_model_valid.acc.ave/test_dev93|503|48634|99.1|0.3|0.5|0.2|1.0|37.8|
+|decode_lm_lm_train_lm_transformer_en_char_valid.loss.ave_asr_model_valid.acc.ave/test_eval92|333|33341|99.4|0.2|0.3|0.1|0.7|33.0|
```

Comment:
Can you add some comments to point to the configuration file used to obtain this result?